modules/assim.batch/R/pda.emulator.R

```diff
@@ -26,9 +26,6 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
                   prior.id=prior.id, chain=chain, iter=iter, adapt=adapt, 
                   adj.min=adj.min, ar.target=ar.target, jvar=jvar, n.knot=n.knot, burnin=burnin)
```

Comment:
I think a good piece of documentation to any function using a `settings` object would be a list of all the pieces of settings required by that function, at least to one level of hierarchy. For example, to make this run, you need at least `settings$assim.batch`, `settings$database`, and `settings$pfts` (possibly one or two more, I didn't do a very comprehensive search), so it would be good to add a comment block to the beginning indicating this.